src/components/RandomBeer.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+
+function RandomBeer(props) {
+
+    const [beersArr, setBeersArr] = useState([]);
```

Comment:
Looking for a single beer

```suggestion
    const [randomBeer, setRandomBeer = useState(null);
```